tracing-subscriber/tests/cached_subscriber_filters_dont_break_other_layers.rs

```diff
@@ -0,0 +1,123 @@
+#![cfg(feature = "registry")]
```

Comment:
i'd rename this file to `tracing-subscriber/tests/cached_subscriber_filters_dont_break_other_subscribers.rs`